site/zh/blog/new-in-devtools-92/index.md

```diff
@@ -37,11 +37,11 @@ Chromium 议题: [1203241](https://crbug.com/1203241)
 
 ## 支持控制台中的 const 重声明 {: #const-redeclaration }
 
-在已有的对[`let` 和 `class` 重声明](/blog/new-in-devtools-80/#redeclarations)的支持之上，控制台现在也支持 const 重声明了。对于使用控制台来试验新 JavaScript 代码的 Web 开发者来说，无法重声明曾是一个常见的困扰。
+控制台现在也支援 const 重声明，在此之前，DevTools 只支援[`let` 和 `class` 的重声明](/blog/new-in-devtools-80/#redeclarations)。无法在控制台中重声明是开发者们常见的困扰，尤其是当您在试验新 JavaScript 代码的时候。
 
-该功能允许开发者将代码复制粘贴进 DevTools 控制台，试运行它，对代码进行小的调整，再重复这个过程，并且不需要刷新页面。在此之前，如果控制台中的代码重声明了一个 `const` 变量，DevTools 会抛出一个语法错误。
+这项新功能允许开发者将代码复制粘贴进 DevTools 控制台，进行测试和调整。您现在可以无限重复这个过程，并且不需要刷新页面。在此之前，如果控制台中的代码重声明了一个 `const` 变量，DevTools 就会抛出一个语法错误。
 
-请参见下面的示例。`const` 重声明的支持是**跨 REPL 脚本**的（参见变量 `a`）。请注意，以下场景是按照设计不被支持的：
+请参见下面的示例。`const` 重声明的支持是**跨 REPL 脚本**的（参见变量 `a`）。按照设计，以下场景是不被支持的：
```

Comment:
Nice! I was trying to translate "note that", but I agree it's better this way since the Chinese version of "note that" feels quite abrupt.